resources_pages/install_ds_stack_windows.md

```diff
@@ -3,290 +3,540 @@ layout: page
+
+To test if the installation was successful open the `SQL Shell` app from the Start menu. You will be asked to setup your configuration, accept the default value (the one within square brackets) for the first four values by pressing enter four times, then type in your password and press enter one last time. It should look like this if it is working correctly:
+
+![](/resources_pages/imgs/psql-windows.png)
+
 ## Docker
 
 You will use Docker to create reproducible, sharable and shippable computing environments for your analyses. For this you will need a Docker account. You can sign up for a free one [here](https://store.docker.com/signup?next=%2F%3Fref%3Dlogin).
 
-After signing-up and signing into the Docker Store, go [here](https://store.docker.com/editions/community/docker-ce-desktop-windows) and click on the "Get Docker" button on the right hand side of the screen. Then follow the installation instructions on that screen.
+After signing-up and signing into the Docker Store, go [here](https://store.docker.com/editions/community/docker-ce-desktop-windows) and click on the "Get Stable" button on the right hand side of the screen. Then follow the installation instructions on that screen.
 
-After installation, restart Powershell and type the following to ask for the version:
+After installation (Docker will make you sign out to finish installing), launch a terminal and type
 
 ```
 docker run hello-world
 ```
 
-you should see something like this if you were successful:
+which should output something like this:
 
 ```
 Unable to find image 'hello-world:latest' locally
 latest: Pulling from library/hello-world
-c04b14da8d14: Pull complete
-Digest: sha256:0256e8a36e2070f7bf2d0b0763dbabdd67798512411de4cdcf9431a1feb60fd9
+0e03bdcc26d7: Pulling fs layer
+0e03bdcc26d7: Verifying Checksum
+0e03bdcc26d7: Download complete
+0e03bdcc26d7: Pull complete
+Digest: sha256:49a1c8800c94df04e9658809b006fd8a686cab8028d33cfba2cc049724254202
 Status: Downloaded newer image for hello-world:latest
-
 Hello from Docker!
+This message shows that your installation appears to be working correctly.
+To generate this message, Docker took the following steps:
+ 1. The Docker client contacted the Docker daemon.
+ 2. The Docker daemon pulled the "hello-world" image from the Docker Hub.
+    (amd64)
+ 3. The Docker daemon created a new container from that image which runs the
+    executable that produces the output you are currently reading.
+ 4. The Docker daemon streamed that output to the Docker client, which sent it
+    to your terminal.
+To try something more ambitious, you can run an Ubuntu container with:
+ $ docker run -it ubuntu bash
+Share images, automate workflows, and more with a free Docker ID:
+ https://hub.docker.com/
+For more examples and ideas, visit:
+ https://docs.docker.com/get-started/
 ```
 
+### Customizing VS Code with Extensions
```

Comment:
Should we move this to the vscode section? It is not needed for anything else I think so breaks the flow, but also clearly belons to vscode.

Reply:
I initially had it higher up, but these extensions work with things like Git, Python and Docker, which are installed later than VS Code, so that we can use VS Code as an editor when installing and configuring things. Therefore I moved them to the end and believe they should stay there.